test/powershell/Language/Scripting/NativeExecution/NativeCommandProcessor.Tests.ps1

```diff
@@ -347,3 +347,68 @@ Describe "Run native command from a mounted FAT-format VHD" -tags @("Feature", "
         $result | Should -BeExactly $expected
     }
 }
+
+Describe "Scriptblock passed as arguments to native executables" -tag @("CI") {
+    BeforeAll {
+        $skipTest = $true
+        if (Get-Command PowerShell) {
+            $skipTest = $false
+        }
+
+        [bool]$scriptblockpassingFeature = (Get-ExperimentalFeature PSNativeScriptBlockArgument).Enabled
+    }
+
+    # current behavior hasn't changed for PowerShell
+    Context "Scriptblocks are encoded when calling PowerShell or pwsh" {
+        It  "A direct call to 'pwsh -command {get-date}' is encoded correctly" {
+            pwsh -command {get-date} | Should -BeOfType DateTime
```

Comment:
Please use exact path as we do in our tests:
```powershell
            $pwsh = "$PSHOME\pwsh"
```

Reply:
I did this purposefully to validate the symlink path. There is a slight difference in the code paths for this execution which I believe should be validated. I will _add_ a test with this expression, but I want to keep this.

```powershell
PS> trace-command -pshost -name parameterbinding { & "$PSHOME/pwsh" -c { get-date } }
DEBUG: 2022-08-26 12:25:49.6935 ParameterBinding Information: 0 : BIND NAMED native application line args [/usr/local/Cellar/powershell-preview/7.3.0-preview.3/libexec/pwsh]
DEBUG: 2022-08-26 12:25:49.6936 ParameterBinding Information: 0 :     BIND cmd line arg [-encodedCommand] to position [0]
DEBUG: 2022-08-26 12:25:49.6937 ParameterBinding Information: 0 :     BIND cmd line arg [IABnAGUAdAAtAGQAYQB0AGUAIAA=] to position [1]
DEBUG: 2022-08-26 12:25:49.6938 ParameterBinding Information: 0 :     BIND cmd line arg [-inputFormat] to position [2]
DEBUG: 2022-08-26 12:25:49.6938 ParameterBinding Information: 0 :     BIND cmd line arg [xml] to position [3]
DEBUG: 2022-08-26 12:25:49.6938 ParameterBinding Information: 0 :     BIND cmd line arg [-outputFormat] to position [4]
DEBUG: 2022-08-26 12:25:49.6938 ParameterBinding Information: 0 :     BIND cmd line arg [xml] to position [5]
DEBUG: 2022-08-26 12:25:49.7043 ParameterBinding Information: 0 : CALLING BeginProcessing

Friday, August 26, 2022 12:25:51 PM

PS> trace-command -pshost -name parameterbinding { pwsh -c { get-date } }            
DEBUG: 2022-08-26 12:26:20.7768 ParameterBinding Information: 0 : BIND NAMED native application line args [/usr/local/bin/pwsh]
DEBUG: 2022-08-26 12:26:20.7768 ParameterBinding Information: 0 :     BIND cmd line arg [-encodedCommand] to position [0]
DEBUG: 2022-08-26 12:26:20.7768 ParameterBinding Information: 0 :     BIND cmd line arg [IABnAGUAdAAtAGQAYQB0AGUAIAA=] to position [1]
DEBUG: 2022-08-26 12:26:20.7768 ParameterBinding Information: 0 :     BIND cmd line arg [-inputFormat] to position [2]
DEBUG: 2022-08-26 12:26:20.7769 ParameterBinding Information: 0 :     BIND cmd line arg [xml] to position [3]
DEBUG: 2022-08-26 12:26:20.7769 ParameterBinding Information: 0 :     BIND cmd line arg [-outputFormat] to position [4]
DEBUG: 2022-08-26 12:26:20.7769 ParameterBinding Information: 0 :     BIND cmd line arg [xml] to position [5]
DEBUG: 2022-08-26 12:26:20.7845 ParameterBinding Information: 0 : CALLING BeginProcessing
```